source/_static/crd.html

```diff
@@ -15,6 +16,7 @@
   <div categories="" channels="" class="row-even attribute-details">
     <div class="arg">
       <div class="argdef">
+        <div class="attributeName"></div>
```

Comment:
I'm unable to tell what this does. The places where it has previously been used it's been called `<div class="attribute-name">`. It's probably something different but it was very conspicuous.